src/govuk/components/exit-this-page/exit-this-page.mjs

```diff
@@ -115,7 +115,9 @@ ExitThisPage.prototype.exitPage = function (e) {
   // Blank the page
   this.$overlay = document.createElement('div')
   this.$overlay.className = 'govuk-exit-this-page__overlay'
+  this.$overlay.setAttribute('role', 'alert')
   document.body.appendChild(this.$overlay)
+  this.$overlay.innerText = 'Exiting page'
```

Comment:
Not sure how much of a difference this makes, but setting `innerText` before doing the `appendChild` feels neater to me—like we're doing all the set up and then adding the entire overlay element (text nodes and all) to the DOM, rather than adding the element, then updating the `innerText` separately.

Reply:
Ah, never mind! It just hit me that `role="alert"` is a live region, and will only read out content when it's _updated_, so setting `innerText` after-the-fact is kind of necessary. Y'all're good. 👍